pkg/bucketaws/setup.go

```diff
@@ -53,10 +53,10 @@ func ProcessDate(fromDate time.Time) []string {
 // Reading from file syncron.yaml
 func ConfigRead() {
 
-	viper.SetConfigFile("syncron.yaml")
-	//viper.SetConfigType("yaml")
-	viper.AddConfigPath("./config")
-	//viper.AddConfigPath(".")
+	viper.SetConfigName("syncron")
+	viper.SetConfigType("yaml")
+	viper.AddConfigPath(".config/")
```

Comment:
I don't know much about viper, but wouldn't this still be pointing to the working directory? Something like `$HOME/.config/` or similar should be used, unless viper does by default include the user home directory

Reply:
It does it by default yes.

Reply:
TIL, then it should be all good